pkg/utils/scheduling/scheduling.go

```diff
@@ -17,26 +17,8 @@ package scheduling
 import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-var (
-	IgnoredOwners = []schema.GroupVersionKind{
-		{Group: "apps", Version: "v1", Kind: "DaemonSet"},
-	}
-)
-
-func IsOwnedByDaemonSet(pod *v1.Pod) bool {
-	for _, ignoredOwner := range IgnoredOwners {
-		for _, owner := range pod.ObjectMeta.OwnerReferences {
-			if owner.APIVersion == ignoredOwner.GroupVersion().String() && owner.Kind == ignoredOwner.Kind {
-				return true
-			}
-		}
-	}
-	return false
-}
-
```

Comment:
Can we move these other helpers, too?

Reply:
I wanted to keep them separate since one was scheduling and one was just about pod states.